lib/concentrate/pipeline/stop_time_updates_pipeline.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Concentrate.Pipeline.StopTimeUpdatesPipeline do
+  alias Concentrate.Pipeline
+
+  @type opts :: [
+          trip_updates_url: String.t()
+        ]
+
+  @spec pipeline(opts()) :: list()
+  def pipeline(opts) do
+    [source(opts), consumer()]
+  end
+
+  def source(opts) do
+    Pipeline.source(
+      :trip_updates_enhanced,
+      opts[:trip_updates_url],
+      Concentrate.Parser.GTFSRealtimeEnhanced
+    )
+  end
```

Comment:
Will this behave well if the `trip_updates_url` isn't set?

Reply:
No. I was thinking that isn't something we have a need to worry about—is there something I'm missing?

Reply:
It might cause problems to any non-MBTA users since they won't be able to give a trip_updates_url that has the format we expect, but wouldn't make a difference to us.